### PR TITLE
ROX-26446: Add start timestamp field

### PR DESCRIPTION
**Pull request by lvalerom**
### Description

This pulls in the field `StaredTimestamp` of a Scan resource. This is needed to match with the CheckResult new annotation `compliance.openshift.io/last-scanned-timestamp`.

### User-facing documentation

- [x] CHANGELOG is updated **OR** update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [x] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [x] CI results are inspected

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [x] modified existing tests

#### How I validated my change

- [x] Manual testing. Started a OCP cluster running CO and checked the new fields are populated in centraldb.

### CODE REVIEW

File: proto/storage/compliance_operator_v2.proto [hunk 1]
```diff
@@ -213,6 +213,7 @@ message ComplianceOperatorScanV2 {
   NodeRole node_selector = 9;
   ScanStatus status = 10;
   google.protobuf.Timestamp created_time = 11;
+  google.protobuf.Timestamp last_started_time = 17; // @gotags: search:"Compliance Scan Last Started Time,hidden"
```
Comment (clickboo) [inline proto/storage/compliance_operator_v2.proto:216]: A comment to explain the difference between last started time and last executed time would be great.

File: proto/internalapi/central/compliance_operator.proto [hunk 2]
```diff
@@ -235,6 +235,7 @@ message ComplianceOperatorScanStatusV2 {
   int64 current_index = 4;
   string warnings = 5;
   int64 remaining_retries = 6;
+  google.protobuf.Timestamp last_start_time = 9;
```
Comment (dashrews78) [inline proto/internalapi/central/compliance_operator.proto:238]: When the proto engine was updated it was decided (I think) that we should keep fields in numerical order to make it easier to determine what is next.  So recommend you do that here and elsewhere.

File: proto/storage/compliance_operator_v2.proto (hunk 1 above)
Comment (dashrews78) [reply proto/storage/compliance_operator_v2.proto:216]: See comment above on numerical order.

Comment (lvalerom) [reply proto/storage/compliance_operator_v2.proto]: Done

File: proto/internalapi/central/compliance_operator.proto (hunk 2 above)
Comment (lvalerom) [reply proto/internalapi/central/compliance_operator.proto]: Done

**Review by dashrews78 — APPROVED** [review]

**Pull request merged**